Formula/landscaper.rb

```diff
@@ -0,0 +1,37 @@
+class Landscaper < Formula
+  desc "Manage the application landscape in a Kubernetes cluster"
+  homepage "https://github.com/Eneco/landscaper"
+  url "https://github.com/Eneco/landscaper.git",
+      :tag => "1.0.7",
```

Comment:
Found out that the Makefile uses git to set some variables [like VERSION](https://github.com/Eneco/landscaper/blob/master/Makefile#L9).
So I think, we should build from a git checkout.